dpnp/dpnp_utils/dpnp_utils_linearalgebra.py

```diff
@@ -364,7 +366,6 @@ def _gemm_batch_matmul(exec_q, x1, x2, res):
         )
         _manager.add_event_pair(ht_ev, blas_ev)
 
-    res_shape = res.shape
     _, res_is_c_contig, res_is_f_contig = _define_contig_flag(res)
     if row_major:
```

Comment:
Do we need to call `return dpnp.ascontiguousarray(res)` below here, keeping in mind use case when `out` memory is used?
Or probably we need to move that call outside `_gemm_batch_matmul`, for instance somewhere to the end of `dpnp_matmul`?